api-functions/cms/ContentfulReporting.ts

```diff
@@ -0,0 +1,41 @@
+import { gql } from "graphql-request";
+import { ReportingContent } from "../../interfaces/contentful-reporting";
+import { client, CMS_ERRORS } from "./ContentfulLanding";
+
+const reportingQuery = gql`
+  query AboutUsPageContent {
+    entryCollection(
+      where: {
+        contentfulMetadata: { tags: { id_contains_all: ["reporting"] } }
+      }
+    ) {
+      items {
+        __typename
+        ... on RichTextOnly {
+          title
+          richDescription: description {
+            json
+          }
+        }
+      }
+    }
+  }
+`;
+
+export async function fetchReporting() {
+  try {
+    const data = await client.request(reportingQuery);
```

Comment:
These queries are missing their corresponding GraphQL type definitions. Look to the Landing page for an example of this.

```typescript
    const data = await client.request<GraphQLContentfulLandingPageContentQuery>(
      landingPageQuery
    );
```